helios-client/src/main/java/com/spotify/helios/client/DefaultRequestDispatcher.java

```diff
@@ -183,7 +183,15 @@ private HttpURLConnection connect(final URI uri, final String method, final byte
         }
       }
     } catch (Exception e) {
-      log.warn("Couldn't get identities from ssh-agent", e);
+      log.debug("Couldn't get identities from ssh-agent", e);
+      log.warn(
+          "Your Helios cluster is setup to work over HTTPS based on the Helios DNS records "
```

Comment:
Yes. I should be clearer about why using HTTPS, which is good in and of itself for encryption, causes the CLI to try to get your ssh-agent identities, which is needed for authentication which is hopefully done over an encrypted channel.